### ISSUE TITLE
The regression error of predictor is very high

### ISSUE THREAD

**Issue by YirongWho** [facebookresearch/vjepa2#104]
Hello ! I am using the pretrained model downloaded from [huggingface repo](https://huggingface.co/facebook/vjepa2-vitl-fpc64-256).  I am surprised to find that the L1 regression error of the predictor is very high. Am I using the model in a wrong way?
I am using the following code:
```import torch
from decord import VideoReader, cpu
import numpy as np
from transformers import AutoVideoProcessor, AutoModel
from transformers import VJEPA2Model, VJEPA2VideoProcessor

hf_repo = "/my_local_dir/vjepa2-vitl-fpc64-256"
video_url = "my_local_dir/kinetics-mini/val/archery/-Qz25rXdMjE_000014_000024.mp4"

model = AutoModel.from_pretrained(hf_repo)
processor = AutoVideoProcessor.from_pretrained(hf_repo)

vr = VideoReader(video_url, ctx=cpu(0))
frame_idx = np.arange(0, 64) # choosing some frames. here, you can define more complex sampling strategy
video = vr.get_batch(frame_idx).asnumpy() # (T, H, W, C)
video = np.transpose(video, (0, 3, 1, 2)) # transform to (T, C, H, W)

inputs = processor(video, return_tensors="pt")
if torch.cuda.is_available():
    model = model.cuda()
    inputs = inputs.to(model.device)

with torch.no_grad():
    outputs = model(**inputs, context_mask=None, target_mask = None)

target_outputs = outputs.predictor_output.target_hidden_state

predictor_outputs = outputs.predictor_output.last_hidden_state

loss_fct = torch.nn.L1Loss(reduction='mean')
loss = loss_fct(predictor_outputs, target_outputs)
print("The loss between predictor and encoding outputs on masked patches:", loss.item())
```
The results of running this code is : "The loss between predictor and encoding outputs on masked patches: 1.861750602722168"

**Comment by YirongWho** [comment]
I've reviewed the training code, and it seems necessary to apply layer normalization to the teacher network's features before calculating the L1 loss. Since both my teacher and student networks are pre-trained models, I should use:
```
target_outputs = outputs.predictor_output.target_hidden_state
#  add layer norm
target_outputs = torch.nn.functional.layer_norm(target_outputs, target_outputs.shape[-1:])
predictor_outputs = outputs.predictor_output.last_hidden_state

loss_fct = torch.nn.L1Loss(reduction='mean')
loss = loss_fct(predictor_outputs, target_outputs)
print("The loss between predictor and encoding outputs on masked patches:", loss.item())
```
This time the loss is 0.52582186460495.
